### PR TITLE
Small NT melee update and rebalance

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

Some changes and additions to NT melee combat gear

- The Longsword damge has been reduced to 30 from 33 to lessen the abundance of very powerful melee weapons a bit

- The Sword of Truth has had it's damage increased to 33 from 26 (it is now no longer outshone by the longsword). In addition, the special attack now makes victims lose 25 from all stats for 45 seconds. A fair enough tradeoff for losing the OG flashbang stun, I'd say.

- The damage of the halberd has been increased to 33 from 26, because it is a very large weapon and to realistically use it, you have to sacrifice your back slot or hold it in your hands all the time, so if it really is supposed to be a weapon for emergencies, it has to deal enough damage to warrant usage of it. The halberd also accepts 2 less toolmods, to emphasize the use for emergencies instead of a an everyday swagged out weapon.

- The shortsword had it's on-handed damage increased from 20 to 25, but the wielded damage remains the same at 26, to give it an edge over a common cleaver.

- The scourge has been reworked. It now deals 26 damage with considerable AP and deals 15 agony damage when not activated. Activated, it's damage is lowered to 15, but it deals 45 agony damage and gets very high AP. In addition, people attacked by the scourge will scream for mercy, as god wills it.

- Added the NT pilum, a new weapon made from common materials. It can be printed from the shield disk for obvious reasons.
It deals 24 damage onehanded and 26 damage wielded (use it with a shield) and has very high AP. When thrown it deals 45 damage and is nearly guaranteed to embed. However, when it embeds the speartip breaks, which makes it very weak until repaired with a welder.

- The NT shield now has three leather straps on it's back that can be used to store small NT weapons and pilums, to furthermore increase the synergy of shield and spear.

- All NT weapons now have a lower chance of embedding in their targets when used by someone with a cruciform. The NT ritual knife has a much bigger chance of embedding when used by someone without a cruciform- heretics beware!

- The Svallin, Martyr, NT dagger and cahors bottles now fit into the NT tacbelt

- The sheath now fits onto the suit slot, for historical accuracy and in order to give sheathes some actual use

- The NT names have been renamed to those of their roman counterparts.

## Changelog
:cl:
add: Added the NT pilum, printable from the shield disk.
tweak: reworked the scourge to have a weak agony-inducing mode and a strong armor-piercing mode.
balance: rebalanced most NT melee weapons
tweak: the NT tacbelt can now store more NT-related items
tweak: the sheath now fits onto the suit slot.
tweak: NT weapons are less likely to embed when used by someone with a cruciform
tweak: the Sword of truth special attack now causes a short stat decrease
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/nt_melee.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
-//Warning! If you change icon_state or item_state, make sure you change path for sneath as well. icons/obj/sneath.dmi
-/obj/item/weapon/tool/sword/nt
-	name = "NT Shortsword"
+//Warning! If you change icon_state or item_state, make sure you change path for sheath as well. icons/obj/sheath.dmi
+/obj/item/weapon/tool/sword/nt // not supposed to be in the game, had to make the shortsword its own type to prevent fucking up the scourge. sorry.
+	name = "NT Sword"
 	desc = "A saint looking sword, made to do God's work."
 	icon = 'icons/obj/nt_melee.dmi'
 	icon_state = "nt_shortsword"
```
Comment (Evankhell561) [inline code/game/objects/items/weapons/nt_melee.dm:6]: if it shouldn't be in the game, add this:
```suggestion
	icon_state = "nt_shortsword"
	bad_type = /obj/item/weapon/tool/sword/nt
```

**Review by Evankhell561 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/nt_melee.dm [hunk 2]
```diff
@@ -13,20 +14,43 @@
...
+	else
+		embed_mult = initial(embed_mult)
+
+/obj/item/weapon/tool/sword/nt/shortsword
+	name = "NT Gladius"
+	desc = "A saint looking sword, made to do God's work."
+	icon = 'icons/obj/nt_melee.dmi'
+	icon_state = "nt_shortsword"
+	item_state = "nt_shortsword"
+	force = WEAPON_FORCE_DANGEROUS * 1.25
```
Comment (TheShown911) [inline code/game/objects/items/weapons/nt_melee.dm:30]: I don't like modifying force defines like this, i would put the flat damage value and `(WEAPON_FORCE_DANGEROUS * 1.25)` as a comment beside it.

File: code/game/objects/items/weapons/nt_melee.dm [hunk 3]
```diff
@@ -13,20 +14,43 @@
...
+
+
 /obj/item/weapon/tool/sword/nt/longsword
-	name = "NT Longsword"
+	name = "NT Spatha"
 	desc = "A saint looking longsword, recommended by experianced crusaders."
 	icon_state = "nt_longsword"
 	item_state = "nt_longsword"
-	force = WEAPON_FORCE_BRUTAL
+	force = WEAPON_FORCE_DANGEROUS * 1.5
```
Comment (TheShown911) [inline code/game/objects/items/weapons/nt_melee.dm:45]: Ditto.

File: code/game/objects/items/weapons/nt_melee.dm [hunk 4]
```diff
@@ -96,19 +133,111 @@
...
+		target.stun_effect_act(stun, agony, hit_zone, src)
+		O.say(pick("LORD", "MERCY", "SPARE", "ME", "HAVE", "PLEASE"))
+
+/obj/item/weapon/tool/sword/nt/spear
+	name = "NT Pilum"
+	desc = "A saint looking short spear, designed for use with a shield or as a throwing weapon. The spear-tip usually breaks after being thrown at a target, but it can be welded into shape again."
+	icon_state = "nt_spear"
+	item_state = "nt_spear"
+	wielded_icon = "nt_spear_wielded"
+	force = WEAPON_FORCE_DANGEROUS * 1.2
```
Comment (TheShown911) [inline code/game/objects/items/weapons/nt_melee.dm:149]: Ditto.

File: code/game/objects/items/weapons/sword_of_truth.dm [hunk 5]
```diff
@@ -4,6 +4,7 @@
 	icon = 'icons/obj/faction_item.dmi'
 	icon_state = "nt_sword_truth"
 	item_state = "nt_sword_truth"
+	force = WEAPON_FORCE_BRUTAL
```
Comment (TheShown911) [inline code/game/objects/items/weapons/sword_of_truth.dm:7]: May conflict with https://github.com/discordia-space/CEV-Eris/pull/5956

File: code/game/objects/items/weapons/nt_melee.dm (hunk 2 above)
Comment (Alliostra) [reply code/game/objects/items/weapons/nt_melee.dm:30]: https://github.com/discordia-space/CEV-Eris/pull/5918#issuecomment-777182806

File: code/game/objects/items/weapons/sword_of_truth.dm (hunk 5 above)
Comment (Alliostra) [reply code/game/objects/items/weapons/sword_of_truth.dm:7]: I don't see any conflicts showing up in the github, does that mean it's good?

**Comment by drexample** [comment]
This has been here since Feb and has unresolved conflicts. If the author is willing to continue working on this, I'll reopen.

File: code/game/objects/items/weapons/nt_melee.dm [hunk 6]
```diff
@@ -96,19 +133,111 @@
...
+	force = 24
+	force_wielded = WEAPON_FORCE_ROBUST
+	var/tipbroken = FALSE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK | SLOT_BELT 
+	throwforce = 75 
+	armor_penetration = ARMOR_PEN_HALF
+	throw_speed = 3
+	price_tag = 150
+	matter = list(MATERIAL_BIOMATTER = 10, MATERIAL_STEEL = 5,) // easy to mass-produce and arm the faithful
```
Comment (Firefox13) [inline code/game/objects/items/weapons/nt_melee.dm:158]:
```suggestion
	matter = list(MATERIAL_BIOMATTER = 10, MATERIAL_STEEL = 5) // easy to mass-produce and arm the faithful
```

File: code/game/objects/items/weapons/nt_melee.dm [hunk 7]
```diff
@@ -96,19 +133,111 @@
...
+	desc = "A saint looking short spear, designed for use with a shield or as a throwing weapon. The spear-tip usually breaks after being thrown at a target, but it can be welded into shape again."
+	icon_state = "nt_spear"
+	item_state = "nt_spear"
+	wielded_icon = "nt_spear_wielded"
+	force = 24
+	force_wielded = WEAPON_FORCE_ROBUST
+	var/tipbroken = FALSE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK | SLOT_BELT 
+	throwforce = 75
```
Comment (Firefox13) [inline code/game/objects/items/weapons/nt_melee.dm:154]: wtf

Comment (Alliostra) [reply code/game/objects/items/weapons/nt_melee.dm:154]: throwforce does not translate to actual damage, which is throwforce*0.6. And yes, I tested it, it deals 45 damage when thrown.

**Comment by Alliostra** [comment]
don't merge until further notice, The only reason why it doesn't have conflicts anymore is that there are no more sprites

File: code/game/objects/items/weapons/sword_of_truth.dm [hunk 8]
```diff
@@ -63,13 +64,25 @@
 				var/obj/item/weapon/implant/core_implant/I = M.get_core_implant(/obj/item/weapon/implant/core_implant/cruciform)
 				if(I && I.active && I.wearer)
 					continue
+				M.stats.addTempStat(STAT_TGH, -STAT_LEVEL_ADEPT, 45 SECONDS)
+				M.stats.addTempStat(STAT_VIG, -STAT_LEVEL_ADEPT, 45 SECONDS)
+				M.stats.addTempStat(STAT_ROB, -STAT_LEVEL_ADEPT, 45 SECONDS)
+				M.stats.addTempStat(STAT_BIO, -STAT_LEVEL_ADEPT, 45 SECONDS)
+				M.stats.addTempStat(STAT_COG, -STAT_LEVEL_ADEPT, 45 SECONDS)
+				M.stats.addTempStat(STAT_MEC, -STAT_LEVEL_ADEPT, 45 SECONDS)
```
Comment (Evankhell561) [inline code/game/objects/items/weapons/sword_of_truth.dm:72]: two things, 1 the flashbag proc already reduces the stats of the affected person.
2. specifies the source of the temporary stats, this is done by the fourth argument of the addTempStat proc.

**Review by Evankhell561 — CHANGES_REQUESTED** [review]

**Comment by Alliostra** [comment]
Gonna fix this later

**Pull request closed without merging**